src/Transport/SparkPostTransportTrait.php

```diff
@@ -26,11 +26,13 @@
      *
      * @param  ClientInterface $client
      * @param  string $key
+     * @param  array $options
      */
-    public function __construct(ClientInterface $client, $key)
+    public function __construct(ClientInterface $client, $key, $options)
```

Comment:
Can you change this line to:
`public function __construct(ClientInterface $client, $key, $options = [])`